ngrok/src/session.rs

```diff
@@ -498,9 +498,13 @@ impl SessionBuilder {
         mut self,
         client_type: impl Into<String>,
         version: impl Into<String>,
+        comments: Option<Vec<impl Into<String>>>,
```

Comment:
I think we can leave it as `Option<impl Into<String>>`. The varargs in the Go implementation were just a way to fake optional arguments without having to resort to nullable pointers or checking for an empty string.

Reply:
Pushed a commit with this change, and also speculatively renamed to `client_info` and removed the doc hiding.